pkg/traceql/engine.go

```diff
@@ -165,8 +165,8 @@ func (e *Engine) asTraceSearchMetadata(spanset Spanset) *tempopb.TraceSearchMeta
 	return metadata
 }
 
-func unixMilliToNano(ts uint32) uint64 {
-	return uint64(ts) * 1000
+func unixSecToNano(ts uint32) uint64 {
+	return uint64(ts) * 1_000_000_000
```

Comment:
nit: there are constants in the `time` package to add some semantics to conversions like this.

I believe it would be `ts * time.Second / time.Nanosecond` since you are translating seconds -> nanoseconds. Dunno if that reds better or worse :).

Reply:
Yeah, that's fair. It's not super compact due to type casting but it's more intent revealing.